### PR TITLE
add view template upload projects

### PR DESCRIPTION
**Pull request by ruddy-m**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/java-specific/introduction-to-spring-boot/view-template/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- rest-api
+- rest-templates
+title: Spring Boot - View Template with Thymeleaf and Upload Files
+---
+
+Java is mostly used for backend application but sometimes you might need to harnes javascript skill within java to run some front application that does something that mostly is backend heavy, this is what we are going to do here
+
+In this project we are going to upload a file from the frontend and process it in the backend
+
+#### Step 1
```
Comment (sheenarbw) [inline content/projects/java-specific/introduction-to-spring-boot/view-template/_index.md]: Are you sure all the heading levels make sense? Step 1 is <h4> and step 2 is <h3>

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Can you please add this to the right place in the java syllabus? If a syllabus doesn't directly or indirectly require something then it doesn't end up on the learners' boards.

File: content/projects/java-specific/introduction-to-spring-boot/view-template/_index.md (hunk 1 above)
Comment (ruddy-m) [reply content/projects/java-specific/introduction-to-spring-boot/view-template/_index.md]: thank you, fixed

**Comment by ruddy-m** [comment]
> 

Which place is the right place I added it in this directory `http://localhost:1313/projects/java-specific/introduction-to-spring-boot/view-template/ `which i think makes sense am I missing something

**Comment by sheenarbw** [comment]
> > 
> 
> Which place is the right place I added it in this directory `http://localhost:1313/projects/java-specific/introduction-to-spring-boot/view-template/ `which i think makes sense am I missing something

Take a look at this directory:
```
content/syllabuses
```

When we add a learner to Tilde then we register them for a bunch of courses by associating the learner with these syllabuses in whatever order makes sense (many to many relationships).

If a learner is associated with a syllabus then they have to do all the things listed in the syllabus page in the order listed there, as well as all the prerequisites for all the listed items.  

Eg: if project X has 2 parts, part 1 and part 2 such that part 1 is a prerequisite for part 2 then:
- adding part 1 to a syllabus means that the learners just needs to do part 1
- adding part 2 to a syllabus means that the learners need to do both parts

So for this project you just made, nothing uses it as a prerequisite so it needs to be explicitly added to the java syllabuses. It probably just needs to be mentioned in  `content/syllabuses/java-part-2.md`

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**